src/main/app/case/CaseApi.ts

```diff
@@ -95,6 +96,16 @@ export class CaseApi {
     }
   }
 
+  public async getCaseUserRoles(caseId: string, userId: string): Promise<CaseAssignedUserRoles> {
```

Comment:
should it not be called after the user logs in?

Reply:
Yeah that is the plan 👍